test/cypress/e2e/activity.spec.js

```diff
@@ -124,44 +137,53 @@ describe('Dashboard Activity', () => {
     describe(testSet.name, () => {
       beforeEach(() => {
         cy.autologin(accounts.delegate.passphrase, networks.devnet.node);
+        testSet.open();
+        waitBeforeChangeTabAfterLoading();
+        cy.get(ss.delegateStatisticsTab).click();
       });
       describe('Delegate statistics tab', () => {
+        /**
+         * Shows voted delegate's nickname not addresses
+         * @expect delegate's nickname shown
+         */
         it('Shows voted delegate nickname ', () => {
-          testSet.open();
-          waitBeforeChangeTabAfterLoading();
-          cy.get(ss.delegateStatisticsTab).click();
-          cy.get(ss.votedAdress).eq(0).should('have.text', 'genesis_17 ');
+          cy.get(ss.votedAddress).eq(0).should('have.text', 'genesis_17 ');
         });
 
+        /**
+         * Click on voted delegate leads to account page
+         * @expect corresponding delegate name is shown on account's page
+         */
         it('Click on voted delegate leads to account page', () => {
-          testSet.open();
-          waitBeforeChangeTabAfterLoading();
-          cy.get(ss.delegateStatisticsTab).click();
-          cy.get(ss.votedAdress).eq(0).click();
+          cy.get(ss.votedAddress).eq(0).click();
           cy.get(ss.delegateName).should('have.text', 'genesis_17');
         });
 
+
         // TODO Fix after corresponding bugfix
+        /**
+         * Shows who-voted-for-this-delegate's nickname if it was delegate
+         * @expect voters nickname shown
+         */
         xit('Shows voters nickname if it is delegate', () => {
-          testSet.open();
-          waitBeforeChangeTabAfterLoading();
-          cy.get(ss.delegateStatisticsTab).click();
           cy.get(ss.voterAdress).eq(0).should('have.text', 'genesis_1 ');
         });
 
+        /**
+         * Shows who-voted-for-this-delegate's address if it was not-delegate
+         * @expect voters address shown
+         */
```

Comment:
as this is a comment you can remove the dash, and maybe if possible put something more descriptive